package.json

```diff
@@ -64,6 +65,6 @@
   ],
   "dependencies": {
     "@aws-sdk/credential-provider-node": "^3.525.0",
-    "@opensearch-project/opensearch": "^2.5.0"
+    "@elastic/elasticsearch": "^7.17.13"
```

Comment:
Must use the latest: https://www.npmjs.com/package/@elastic/elasticsearch: `8.14.0`

What is `@aws-sdk/credential-provider-node` needed for?

Reply:
Using  `"@elastic/elasticsearch": "^8.14.0"`
Removed `"@aws-sdk/credential-provider-node": "^3.525.0",`

Thanks! Indeed we dont need aws and 8.14.0 is a must for kNN search.